pkgs/applications/science/math/gaviewer/default.nix

```
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl, pkgconfig, antlr, fltk13, libpng12, libGL, libGLU, addOpenGLRunpath }:
+
+stdenv.mkDerivation rec {
+  pname = "gaviewer";
+  version = "0.86";
+
+  meta = with lib; {
+    description = "A tutorial program for geometric algebra";
+    longDescription = ''
+      This is the companion software to the book Geometric Algebra For Computer Science, An Object Oriented Approach to Geometry, published by Morgan Kaufmann Publishers.
+      Most of the figures in the book were generated by GAViewer, a tutorial program for geometric algebra. 
+      They are shots of 2-D or 3-D interactive scenes, and you can increase your understanding of geometric algebra by playing around with them.
+
+      Having GAViewer installed allows you to do more, transcending the book figures:
+      * view results of your own derivations
+      * test correctness of formulas by trying them in various circumstances
+      * generate personal figures for your own geometric algebra presentation (please refer properly to GAViewer if you do)
+    '';
+    homepage = "https://geometricalgebra.org";
+    downloadPage = "https://geometricalgebra.org/gaviewer_download.html";
+    maintainers = with maintainers; [ oliverevans96 ];
+    license = with licenses; gpl2;
```

Do you know if the license is gpl2 or gpl2Plus?

```suggestion
    license = licenses.gpl2;
```